db/schema.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
We talked about it, please not use rubocop for this file. Exclude folder db and config in rubocop.yml

Reply:
I reverted commit that I had made before running rubocop, than I edited rubocop.yml and ran Rubocop again.